lib/common/Memory/SmallFinalizableHeapBlock.cpp

```diff
@@ -2,6 +2,7 @@
 // Copyright (C) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
 //-------------------------------------------------------------------------------------------------------
+#define SMALL_FINALIZABLE_HEAP_BLOCK_CPP
```

Comment:
If you can find an alternative, we can probably avoid the turn-off-PCH-for-these-files stuff in MSVC. But I wasn't able to find anything yet